packages/valibot/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import { type FieldErrors, type Validator, createValidator } from "@rvf/core";
+import {
+  type BaseIssue,
+  type Config,
+  type GenericSchema,
+  type GenericSchemaAsync,
+  type InferIssue,
+  type InferOutput,
+  safeParseAsync,
+} from "valibot";
+
+type MaybeAsyncSchema = GenericSchema | GenericSchemaAsync
+
+function formatIssuePath(issue: BaseIssue<unknown>): string | null {
+  if (!issue.path || issue.path.length === 0) return null;
+
+  let result = '';
+  
+  for (const item of issue.path) {
```

Comment:
We have a helper that should be able to handle this. Could we use that here?

```typescript
import { pathArrayToString } from '@rvf/set-get';

const pathArray = issue.path.map(path => path.key);
const result = pathArrayToString(pathArray);;
```

Reply:
done!

added two test cases from zod tests

can you please check new version of `parseIssues` function? 
now we have some recursion because of union issues

also I want to check if I could find any library-specific quirks to write some tests if needed